README.md

```diff
@@ -73,4 +75,99 @@ Django-major-event-log was developed at the UNT Libraries.
 Contributors:
 
 * [Gio Gottardi](https://github.com/somexpert)
+* [Damon Kelley](https://github.com/damonkelley)
+
+
+Developing/Testing
+-------------------------
+
+#### Requirements ####
+
+- Django == 1.8
+- Python >= 2.7
+
+I suggest doing the following within a virtual environment. To do so, you will
```

Comment:
Setting up the virtual environment is kind of out of scope, but I am okay with it being in here. If you are going to have it though, you might also mention virtualenvwrapper, as that is commonly used.